dev/README.md

```diff
@@ -6,7 +6,7 @@
 
 You will need the following installed locally on your machine to support local development:
 
-* [Minikube v1.16.0](https://minikube.sigs.k8s.io/docs/start/)
+* [Minikube (up to v1.25.2 tested)](https://minikube.sigs.k8s.io/docs/start/)
```

Comment:
Actually we did not test every previous minikube version. Let's suggest only 1.25.2 if you are ok